components/CollapsibleList/icon.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+const SvgContainer = styled.svg`
+  margin-left: auto;
+  transition: transform 0.6s ease;
+  transform: ${(props) =>
+    props.rotateIcon === 'arrow_icon_rotate'
+      ? 'rotate(90deg)'
+      : 'rotate(0deg)'};
+`;
+const Arrow = (props) => {
+  return (
+    <SvgContainer
```

Comment:
I love this SVG container concept. Haven't seen many junior engineers think this way ❤️ .